src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -790,6 +792,7 @@ export class PostgresStorageAdapter implements StorageAdapter {
     this._client = client;
     this._pgp = pgp;
     this.canSortOnJoinTables = false;
+    this.engine = 'PostgreSQL';
```

Comment:
You can get the PG version via `select version()` and parse it out. Add to performInitalization

Reply:
I used `SHOW server_version`